gcp_variant_transforms/transforms/variant_to_bigquery.py

```diff
@@ -113,7 +119,22 @@ def expand(self, pcoll):
             self._allow_incompatible_records,
             self._omit_empty_sample_calls))
     if self._limited_write:
-      bq_rows |= 'LimitWrite' >> limit_write.LimitWrite(_WRITE_SHARDS_LIMIT)
+      bq_row_partitions = bq_rows | beam.Partition(random_partition,
+                                                   _NUM_RANDOM_PARTITIONS)
+      bq_writes = []
+      for i in range(_NUM_RANDOM_PARTITIONS):
+        bq_rows = (bq_row_partitions[i] | 'LimitWrite' + str(i) >>
+                   limit_write.LimitWrite(_WRITE_SHARDS_LIMIT))
+        bq_writes.append(bq_rows | 'WriteToBigQuery' + str(i) >>
```

Comment:
nit: I think the indentation would be nicer if you break after `append(` and then on `|`